src/digital/v2_compat.rs

```diff
@@ -140,10 +144,12 @@ mod tests {
         }
     }
 
+    #[cfg(feature = "unproven")]
```

Comment:
... and this

Reply:
`v2::ToggleableOutputPin` is `unproven`, so this test wouldn't build without the flag anyway.